Init Ver with Admin CLI and community ERC721 cc

wan-yong opened the pull request:
Signed-off-by: Wan Yong <wan_yong@msn.com>

wan-yong committed:
Init Ver with Admin CLI and community ERC721 cc

Signed-off-by: Wan Yong <wan_yong@msn.com>

SamYuan1990 commented:
some question before merge.
- CI plan?
- it seems we only have code for application, any plan for fabric code?
- document for apis ex swagger?

wan-yong committed:
Modified for CI with github Action.

Signed-off-by: Wan Yong <wan_yong@msn.com>

wan-yong committed:
Create main.yml

Signed-off-by: Wan Yong <wan_yong@msn.com>

wan-yong commented:
1. Added CI Actions.
3. Added Chaincode for convenience, which is no change from the community.
5. Swagger for NodeJS is a little more complex than in Spring Boot - Java. Put this in plan, not this Release.

SamYuan1990 reviewed (approved):
LGTM
